lib/ProductOpener/Display.pm

```diff
@@ -3898,6 +3898,11 @@ HTML
 						$user_template_data_ref->{edit_profile} = 1;
 						$user_template_data_ref->{orgid} = $orgid;
 					}
+					my @org_members;
+					foreach my $member_id (keys %{$user_or_org_ref->{members}}) {
```

Comment:
In this loop, you will need to load the user file that contains details such as the e-mail, language, country etc.

In Users.pm and Display.pm, we have code like this to do that:

`my $user_file = "$data_root/users/" . get_string_id_for_lang("no_language", $user_id) . ".sto";

			if (-e $user_file) {
				$user_ref = retrieve($user_file);`

It would be a good thing to create a retrieve_user($user_id) function in Users.pm to do that.